src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -477,67 +473,149 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
                   >
                     <StyledDateField
                       key={datePart}
+                      size={size}
                       isYearInput={datePart.length === 4}
-                      isEnd={isEnd}
                       hasValidationIconInField={
-                        hasValidationIcon &&
-                        !validationOnLabel &&
-                        !validationRedesignOptIn
+                        !validationOnLabel && isEnd && validationInField
                       }
+                      isEnd={isEnd}
                     >
-                      <FormSpacingProvider marginBottom={undefined}>
-                        <Typography
-                          mb="4px"
-                          id={labelId}
-                          data-role="numeral-date-input-text"
-                        >
-                          {getDateLabel(datePart, locale)}
-                        </Typography>
-                        <Textbox
-                          {...(index === 0 && { id: uniqueId })}
-                          disabled={disabled}
-                          readOnly={readOnly}
-                          value={dateValue[datePart as keyof NumeralDateObject]}
-                          onChange={(e) =>
-                            handleChange(e, datePart as keyof NumeralDateObject)
-                          }
-                          onBlur={handleBlur}
-                          ref={(element) => {
-                            refs.current[index] = element;
-                            if (!inputRef) {
-                              return;
-                            }
-                            if (typeof inputRef === "function") {
-                              inputRef(element);
-                            } else {
-                              inputRef.current = element;
-                            }
-                          }}
-                          error={!!internalError}
-                          warning={!!internalWarning}
-                          info={!!info}
-                          {...(isEnd &&
-                            !validationRedesignOptIn &&
-                            !validationOnLabel && {
-                              error: internalError,
-                              warning: internalWarning,
-                              info,
-                            })}
-                          size={size}
-                          tooltipPosition={tooltipPosition}
-                          aria-labelledby={labelId}
-                          required={required}
-                        />
-                      </FormSpacingProvider>
+                      <StyledDateLabel
```

Comment:
Comment: I wonder if this should be a `label` element so that when a user clicks on it the relevant input is focused